app/logic/userManagement.py

```diff
@@ -53,7 +53,7 @@ def changeProgramInfo(newProgramName, newContactEmail, newContactName, newLocati
 def getAllowedPrograms(currentUser):
     """Returns a list of all visible programs depending on who the current user is."""
     if currentUser.isCeltsAdmin:
-        return Program.select().order_by(Program.programName)
+        return Program.select().where(Program.isOtherCeltsSponsored == False).order_by(Program.programName)
```

Comment:
Why are we excluding these programs? The previous query didn't restrict by program at all.